localstripe/resources.py

```diff
@@ -2281,6 +2283,8 @@ def __init__(self, customer=None, metadata=None, items=None,
                            for txr in default_tax_rates)
             if trial_period_days is not None:
                 assert type(trial_period_days) is int
+            if backdate_start_date is not None:
+                assert type(backdate_start_date) is int
```

Comment:
For consistency with other dates validations, it needs `assert backdate_start_date > 1500000000`.